gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/MRJobLauncher.java

```diff
@@ -123,7 +118,9 @@ public MRJobLauncher(Properties jobProps) throws Exception {
   }
 
   public MRJobLauncher(Properties jobProps, Configuration conf) throws Exception {
-    super(jobProps, ImmutableMap.<String, String> of());
+    super(jobProps, FileSystem
```

Comment:
Can you move the creation of the `FileSystem` into its own method (something like `buildFileSystem`), so that the `super` declaration isn't so long.